unit-tests-java8/src/test/java/org/eclipse/collections/test/map/mutable/ordered/MutableOrderedMapTestCase.java

```diff
@@ -22,4 +28,33 @@
 
     @Override
     <K, V> MutableOrderedMap<K, V> newWithKeysValues(Object... elements);
+
+    @Test
+    default void MutableOrderedMap_removeIf()
+    {
+        MutableOrderedMap<Integer, String> map = this.newWithKeysValues(1, "1", 2, "Two", 3, "Three");
+
+        Assert.assertFalse(map.removeIf(Predicates2.alwaysFalse()));
+        Assert.assertEquals(this.newWithKeysValues(1, "1", 2, "Two", 3, "Three"), map);
+        Assert.assertTrue(map.removeIf(Predicates2.alwaysTrue()));
+        Verify.assertEmpty(map);
+
+        map.putAll(Maps.mutable.with(1, "One", 2, "TWO", 3, "THREE", 4, "four"));
+        map.putAll(Maps.mutable.with(5, "Five", 6, "Six", 7, "Seven", 8, "Eight"));
```

Comment:
Here you're calling putAll using a hashtable, so the keys are in random order. This means that you're not asserting anything about ordering.